tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+meta:
+  title: How to setup a Proxmox cluster on Elastic Metal servers using Private Networks
+  description: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+content:
+  h1: How to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+  paragraph: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+categories:
+  - dedicated servers
+  - elastic-metal
+tags: Proxmox Elastic-Metal KVM-over-IP virtual-machine container
+dates:
+  validation: 2022-01-10
+  posted: 2022-01-10
+---
+
+## Proxmox - Overview
+
+[Proxmox Virtual Environment](https://www.proxmox.com/en/proxmox-ve) - or Proxmox VE is a server virtualization platform, based on Debian Linux. It allows the deployment and management of [KVM-based](https://www.linux-kvm.org/page/Main_Page) virtual machines or [LXC](https://linuxcontainers.org/) containers.
+
```

Comment:
In this tutorial, we show you how to install Proxmox on your Elastic Metal servers, configure the Private Network via the Proxmox interface, and create a cluster of virtual machines across the two Elastic Metal servers which communicate via the Private Network. All the virtual machines can be managed from a single Proxmox interface, with the Private Network ensuring a high level of security and low latency.